src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+a,
+a:link,
+a:visited {
+  text-decoration: none;
+}
+
+a:hover {
```

Comment:
```suggestion
a {
```

Reply:
fix it https://github.com/mate-academy/layout_catalog/pull/2887/files#r1138228875